extensions/amp-bind/amp-bind.md

```diff
@@ -176,8 +176,19 @@ When triggered by certain events, `AMP.setState()` also can access event-related
 <input type="range" on="change:AMP.setState({myRangeValue: event.value})">
 ```
 
+#### Modifying history with `AMP.pushState()`
 
- See [Actions and Events in AMP](../../spec/amp-actions-and-events.md) for more details.
```

Comment:
Did this get dropped from the new version?

Reply:
Yea, `AMP.setState` and `AMP.pushState` already links to that file above.